### PR TITLE
Igh trying to fix data submission styling

### PR DESCRIPTION
**Pull request by globusharris**

### CODE REVIEW

File: client/app/views/clientView.js [hunk 1]
```diff
@@ -53,6 +53,15 @@ define(['jquery', 'controllers/clientController', 'controllers/tableController',
     function createResizeSensors(tables) {
       tables.forEach(function (t) {
         var div = $('#' + t.rootElement.id);
+
+        var width = $(window).width();
```
Comment (frederickjansen) [inline client/app/views/clientView.js]: createResizeSensors is never called, so this code never gets triggered

File: client/app/controllers/tableController.js [hunk 2]
```diff
@@ -706,25 +706,34 @@ define(['jquery', 'Handsontable', 'table_template', 'filesaver', 'alertify', 'qt
...
   }
 
-  function updateTableWidth(maxWidth) {
+  function updateInstructionWidth(maxWidth) {
 
-    //$('#instructions').css('width', '1153px');
-    $('#instructions').css('max-width', '1153px');
+    var $instructions = $('#instructions');
+    var $container = $('.container');
+    var $card = $('.card.col-md-10.col-md-offset-1');
```
Comment (frederickjansen) [inline client/app/controllers/tableController.js:713]: This selector is too specific. We might want to switch to a different width or offset at some point.

File: client/app/views/clientView.js (hunk 1 above)
Comment (globusharris) [reply client/app/views/clientView.js]: It's called on line 84?

Comment (globusharris) [reply client/app/views/clientView.js]: It's called on line 84?

Comment (frederickjansen) [reply client/app/views/clientView.js]: Are we looking at the same line?
```//createResizeSensors(tables); THIS FUNCTION BREAKS THINGS I THINK! -IRA```

File: client/app/controllers/tableController.js (hunk 2 above)
Comment (globusharris) [reply client/app/controllers/tableController.js:713]: Good point. Also the column/offset parameters are unnecessary. Fixed this but having trouble pushing changes.

**Pull request merged**